packages/desktop_file_utils.rb

```diff
@@ -0,0 +1,33 @@
+require 'package'
+
+class Desktop_file_utils < Package
+  description 'contains a few command line utilities for working with desktop entries'
+  homepage 'https://www.freedesktop.org/wiki/Software/desktop-file-utils/'
+  version '0.23'
+  license 'GPL-2+'
+  compatibility 'all'
+  source_url 'https://www.freedesktop.org/software/desktop-file-utils/releases/desktop-file-utils-0.23.tar.xz'
+  source_sha256 '6c094031bdec46c9f621708f919084e1cb5294e2c5b1e4c883b3e70cb8903385'
+
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/desktop_file_utilities-0.23-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/desktop_file_utilities-0.23-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/desktop_file_utilities-0.23-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/desktop_file_utilities-0.23-chromeos-x86_64.tar.xz',
```

Comment:
These binaries don't match the package name.  I see what you are trying to do but is it worth it just to shorten the name?  I imagine most of the time you would never explicitly install this package anyway.

Reply:
I did have to explicitly install it once, I forget why, and that's how I noticed that the name changed. I'll change it back in this PR but it shud be changed later